modules/scripts/genindex.py

```diff
@@ -96,14 +96,23 @@ def get_index(root, subprojects, sub_docpaths, javadoc):
     # add javadoc if javadoc exist
     if javadoc != "":
         index = add_javadoc(index)
-
+    if build_dir != "":
+        index = add_subproject(index, build_dir)
     return '\n\n'.join(index) + '\n'
 
 def add_javadoc(index):
     javadoc = ["API Documentation", "\n", "=================", "\n",
      "* `javadoc <./javadoc>`_"]
     return index + javadoc
 
+def add_subproject(index, build_dir):
```

Comment:
Why is this needed. Sub projects were already being included in the generated index